code/modules/clothing/suits/armor.dm

```diff
@@ -142,7 +142,7 @@
 
 /obj/item/clothing/suit/armor/vest/warden
 	name = "Warden's armored jacket"
-	desc = "An armoured jacket with silver rank pips and livery."
+	desc = "An armored jacket with silver rank pips and livery."

```

Comment:
Hurts the British side of me, but the rest of the descriptions don't use the (correct) 'u', so I've changed this to bring it inline.